.travis.yml

```diff
@@ -66,13 +66,13 @@ jobs:
       cache:
         directories:
           - "$HOME/.gem/specs"
-          - ".doc/framework/node_modules"
+          - "./doc/framework/node_modules"
       before_script:
         - npm run doc-prepare
-        - npm run --prefix .doc/framework clone-repos
+        - npm run --prefix ./doc/framework clone-repos
```

Comment:
This won't work due to the migration to `kuzdoc`. I suggest  you merge the `dev-3` branch and keep the implementation coming from that branch.

Reply:
After talking with you, won't do